clang/include/clang/Sema/ParsedAttr.h

```diff
@@ -808,7 +808,7 @@ class AttributePool {
 
 class ParsedAttributesView {
   friend class AttributePool;
-  using VecTy = llvm::SmallVector<ParsedAttr *>;
+  using VecTy = llvm::SmallVector<ParsedAttr *, 2>;
```

Comment:
Curious why '2' was found?  Did you see perf-benefit from 2 over 0(aka, `std::vector`)?  Presumably the benefit of cache locality isn't really applicable as attributes are comparatively rare, so 'while we're at it', perhaps saving twice as much would be valuable?